src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -700,4 +720,116 @@ private void cancelElectionsForReferenceIds(List<String> referenceIds) {
       }
     });
   }
+
+  private void sendDarNewCollectionElectionMessage(List<User> users, String darCode)
```

Comment:
Moving DarCollection business logic from the EmailService to the DarCollectionService.